bootstrap/api/requests.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/mainflux/mainflux/pkg/errors"
 )
 
+const maxLimitSize = 100
```

Comment:
No need for this "Size" I think. `maxLimit` is sufficient. Is there some particular reason you introduced this `Size` suffix?

Reply:
Constency reasons. Also think about `maxNameSize` or any other. Are we going to use `maxName`?

Reply:
Uh... I think it is not exactly the same, but I agree, I would probably keep `maxLimitSize` then. @dusanb94 what do you say?

Reply:
Keeping it makes sense, since Limit is actually name of the param (though limitSize sounds obnoxious).